src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -15,13 +18,22 @@
 {
     @Override
     public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
-    {
+    {   
+        final Player player = getPlayer(args[0]);
```

Comment:
`getPlayer()` is called while `args[0]` might be undefined. This may lead to a `ArrayIndexOutOfBoundsException` when running _/smite_ without arguments.